src/VirtualClient/VirtualClient.Dependencies/GridAndNvidiaDriverInstallation.cs

```diff
@@ -142,6 +104,8 @@ protected override async Task ExecuteAsync(EventContext telemetryContext, Cancel
                                 ErrorReason.LinuxDistributionNotSupported);
                     }
 
+                    await this.HoldKernelPackage(linuxDistributionInfo.LinuxDistribution, telemetryContext, cancellationToken);
```

Comment:
I'm creating a backlog item on HoldKernelPackage. Since our customers may not hold the kernel pkg, we can make holderKernelPkg a bool parameter at the class level.